src/modules/components/member/memberAvatar/memberAvatar.tsx

```diff
@@ -45,9 +45,10 @@ export const MemberAvatar: React.FC<IMemberAvatarProps> = (props) => {
     });
     const resolvedAvatarSrc = avatarSrc ?? ensAvatarData ?? undefined;
 
-    const blockiesSrc = resolvedAddress
-        ? blockies.create({ seed: addressUtils.getChecksum(resolvedAddress), scale: 8, size: 8 }).toDataURL()
-        : undefined;
+    const blockiesSrc =
+        resolvedAddress && !ssrUtils.isServer()
```

Comment:
Why are we forcing blockies to only render clientside?

Reply:
Looks like `blockies` uses `document.createElement` to create a canvas and generate the address image => https://github.com/pedrouid/blockies-ts/blob/master/src/index.ts#L43
This is causing the new app-next repository to crash when we try to render the members avatars on the server side

Reply:
Ah, makes sense, thanks!